README.md

```diff
@@ -205,12 +219,32 @@ class DummyRequest implements CallableRequest
 
    public $abcProperty;
```

Comment:
Örnekler için tanımlalar yaparken _foo_, _bar_ ve _baz_ gibi kelimeler kullanılabilir, bu sayede bir standart oluşturmuş oluruz;

```suggestion
   public $foo;
```